superset/queries/saved_queries/api.py

```diff
@@ -252,3 +265,135 @@ def export(self, **kwargs: Any) -> Response:
             as_attachment=True,
             attachment_filename=filename,
         )
+    @expose("/import/", methods=["POST"])
+    @protect()
+    @safe
+    @statsd_metrics
+    @event_logger.log_this_with_context(
+        action=lambda self, *args, **kwargs: f"{self.__class__.__name__}.import_",
+        log_to_statsd=False,
+    )
+    def import_(self) -> Response:
+        """Import chart(s) with associated datasets and databases
```

Comment:
chart -> saved_query (same thing in other places)

Reply:
embarrassing, thank you for pointing it out.

Reply:
I've done that so many times! 😆